__tests__/app.test.js

```diff
@@ -248,5 +248,66 @@ return request(app)
   expect(res.body.msg).toBe('invalid username')
 
 });
+})
+})
+
+describe('6. PATCH /api/articles/:article_id', () => {
+  test('status:201, respond with updated article object', () => {
+  const voteChange =  {inc_votes: 1}
+  return request(app)
+  .patch('/api/articles/1')
+  .send(voteChange)
+  .expect(201)
+  .then((res) =>{
+      expect(res.body.article).toMatchObject({
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 101
+      })
+  })
+})
+test('status:201, respond with updated article object, for negative votes', () => {
+  const voteChange =  {inc_votes: -200}
+  return request(app)
+  .patch('/api/articles/1')
+  .send(voteChange)
+  .expect(201)
+  .then((res) =>{
+      expect(res.body.article).toMatchObject({
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: -100
+      })
+  })
+})
+test('status:404, for when the article does not exist', () => {
+  const voteChange =  {inc_votes: -200}
+  return request(app)
+  .patch('/api/articles/2000')
+  .send(voteChange)
+  .expect(404)
+  .then((res) =>{
+    expect(res.body.msg).toBe('article not found')
+  })
+  
+})
+test('status:400, for when the article_id is invalid', () => {
+  const voteChange =  {inc_votes: 100000}
+  return request(app)
+  .patch('/api/articles/nonsense')
+  .send(voteChange)
+  .expect(400)
+  .then((res) =>{
+    expect(res.body.msg).toBe('invalid article id')
+  })
+  
```

Comment:
Might want to add another 400 for when there's an incorrect body (or it's missing, e.g. inc_votes is not a number or missing)